java/Alena-Vorokh/src/com/zoolatech/lecture1/tasks/_10/Task_10.java

```diff
@@ -0,0 +1,42 @@
+package com.zoolatech.lecture1.tasks._10;
+
+/* Write a program that simulates the guess a number game.
+The program should generate a random number between 1 and N (N should be read from the input)
+ and the user needs to guess it. If the user enters a number less than guessed, the program needs
+ to output “Larger”. If the user enters a number greater than guessed - the program should output “Smaller”.
+ If the user guesses the number correctly - the program should output “That’s it!” and finish.*/
+
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Task_10 {
+    public static void main(String []args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input max value: ");
+        int N = scanner.nextInt();
+        Random rnd = new Random();
+        int secretNumber = rnd.nextInt(N);
+        // Check
+        // System.out.println("Number = " + secretNumber);
+        System.out.println("Input your variant: ");
+        int yourNumber = scanner.nextInt();
+        while (yourNumber != secretNumber){
+            if (yourNumber > secretNumber){
+                System.out.println("Larger!");
+            }
+            if (yourNumber < secretNumber){
```

Comment:
it might be better to use `else-if` statement here instead of two `if` statements: 
```
if (yourNumber > secretNumber){
...
} else if (yourNumber < secretNumber) {
...
}
```
When you use two if statements both conditions are always checked. If you use if-elseif statement - the second condition is not checked if the first one is true.